packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -1249,7 +1249,10 @@ export default class CashierStore extends BaseStore {
             });
             this.setIsTryTransferSuccessful(true);
         } else {
-            this.setErrorMessage(payment_agent_transfer.error, this.resetPaymentAgentTransfer);
+            this.setErrorMessage(
+                payment_agent_transfer.error?.message || payment_agent_transfer.error,
```

Comment:
```suggestion
                payment_agent_transfer.error?.message ?? payment_agent_transfer.error,
```

Reply:
what's the difference 😅

Reply:
API error message will not be empty or 0. it will either have a message prop with value in it or it won't. if the structure is sent with message in it we want that, otherwise the structure with error is used

Reply:
so these are the possible API responses:
1. `payment_agent_transfer = { error: 'some error message' }`
2. `payment_agent_transfer = { error: { message: 'some error message'  }}`

the code is only handling case 1 currently while API some times sends case 2. so we just need to handle case 2 as well. if message is null for case 2, falling back to case 1 doesn't help since there's no usable message for error prop in this case.

Reply:
`or` will not evaluate falsy value, meanwhile `nullish coalescence` will not evaluate only null or undefined. its more towards practice, if it won't be empty string or 0,  then both operators will work fine

Reply:
I think we should use the correct operator based on the scenario, not just replacing all of our `||` with `??`